### ISSUE TITLE
Can I use this project to retrain the model for a video classification task?

### ISSUE THREAD

**Issue by michaelwithu** [v-iashin/video_features#132]

**Comment by v-iashin** [comment]
yes, you can. just extract the features for the videos for which you have labels and train a simple classifier to do what you want.

the training scripts are not provided in this repo. 

here is a generic tutorial for image classification from pytorch: https://pytorch.org/tutorials/beginner/transfer_learning_tutorial.html#convnet-as-fixed-feature-extractor
